.vscode/settings.json

```diff
@@ -0,0 +1,23 @@
+{
+    "eslint.enable": true,
+    "eslint.format.enable": true,
+    "eslint.validate": [
+        "javascript"
+    ],
+    "[javascript]": {
+        "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+        "editor.codeActionsOnSave": {
+        "source.fixAll.eslint": true
+        }
+    },
+    "[css]": {
+        "editor.defaultFormatter": "vtfn.stylint",
```

Comment:
See above RE `vtfn.stylint` vs `stylelint.vscode-stylelint`

Reply:
Hmm, yeah that's strange!  Are you trying to run the new settings with your old default formatter in place?  Maybe we can pair/screenshare to figure it out.